test/install_test.go

```diff
@@ -483,10 +491,10 @@ func TestLogs(t *testing.T) {
 					if errRegex.MatchString(line) {
 						if knownErrorsRegex.MatchString(line) {
 							// report all known logging errors in the output
-							t.Skipf("Found known error in %s log: %s", name, line)
+							t.Logf("Found known error in %s log: %s", name, line)
```

Comment:
@siggy when a know error was found, `t.Skipf` continued into the next container in the loop, so we weren't checking the log lines after that one.